### ISSUE TITLE
Schema not updating in production mode

### ISSUE THREAD

**Issue by ka-weihe** [cube-js/cube#7877]
When adding a new cube to the schema directory, we have that Cube will automatically update the schema in development mode, but in production mode the new Cube is only picked up on restarts. Is this a bug or a feature?

**Comment by seatunneltrailuser** [comment]
Yes, +1.  Facing the same issue.  We ran an instance with Dev mode, and another instance with non-development mode,.  The Models created thru the UI (from the Dev Mode instance), is not getting listed with the API in the non-development mode, even though the same folders are mounted within the container (/cube/conf).  

Any suggestion to refresh/reload automatically or via any API will greatly help us.   Currently we are managing with restarting the non-development instance whenever a new model is created.

**Comment by igorlukanin** [comment]
Hi @ka-weihe @seatunneltrailuser 👋 

> Is this a bug or a feature?

This is definitely a feature or, better, intended behavior.

In production mode, Cube would not check if the data model files have been updated due to performance reasons. If you'd like to instruct Cube to reload data model files, please use [`repository_factory`[(https://cube.dev/docs/reference/configuration/config#repository_factory) and [`schema_version`](https://cube.dev/docs/reference/configuration/config#schema_version).

**Comment by igorlukanin** [comment]
@ka-weihe @seatunneltrailuser Did my advice above help?
